### DISCUSSION TITLE
Set x,y offset on clipart

### DISCUSSION THREAD

**Discussion by barathi-k** [pubpub-zz/ppInk#268] [Ideas]
Is there a way we could set x,y offset (in percentage or in pixels) when creating clipart? What I see is the image's top left corner appears at the point where I click. 

If there is an offset parameter (on API) We could create something like a pointer clipart and make sure it points to the exact place where we click. 

(I could do something like this 😂)

![Comp 1_2](https://github.com/pubpub-zz/ppInk/assets/55287040/bd4f5535-5bb8-4430-af47-8ab4b1c05bd3)

**Comment by pubpub-zz** [comment]
converted into issue #269 for change traceability
